app/views/lookup_keys/_values.html.erb

```diff
@@ -0,0 +1,23 @@
+<table class="table table-condensed">
+  <thead>
+  <tr>
+    <th colspan='2' class='col-md-5'><%= _('Attribute Type') %> <span class="help-inline"><%= popover("", _("Matcher is a combination of an attribute and its value, if they match, the value below would be provided.<br> You may use any attribute foreman knows about, such as facts etc for example: <code> domain = example.com </code> or <code> is_virtual = true</code>"),
```

Comment:
Lower case 'Type', upper case 'Foreman', and this should probably be rewritten a little (remind me later).